### DISCUSSION TITLE
option to disable new tab going to daily.dev

### DISCUSSION THREAD

**Discussion by pawanvora2017** [dailydotdev/daily#759] [Feature Requests]
I don't like the fact that clicking New Tab takes me to daily.dev. There should be an option to enable / disable such as behavior.

**Comment by idoshamun** [comment]
Unfortunately, the browsers don't provide the API to enable/disable the new tab functionality.
My recommendation is to use our webapp https://app.daily.dev

**Comment by pawanvora2017** [comment]
Thanks. I’ve been doing that after I removed the extension :-)

If it was something I can access in a side panel by clicking on the extension, it’d be awesome. I feel terrible having to uninstall such a useful extension.

Pawan...
